adaptdl/adaptdl/torch/__init__.py

```diff
@@ -119,6 +120,9 @@ def init_process_group(backend,
                                   rank,
                                   world_size)
 
+    # Initialize Context module.
+    adaptdl.torch.data.context_initialize(batch_size=32)
```

Comment:
How do you plan to make `batch_size` available in `init_process_group`?

Reply:
> How do you plan to make `batch_size` available in `init_process_group`?

We were just giving it a default value here for the global Context, the batch_size could be replaced by users when they use it. But thanks for point it out that we can also remove the default value from here, which would be given from the definition of class Context directly. Further, users are still able to give their input value and the default value will be replaced then. 
Please also have a check with our new commit to review the modification.
Many thanks.